cpu/stm32_common/stmclk.c

```diff
@@ -0,0 +1,179 @@
+/*
```

Comment:
how about we rename this file to `stmclk_f247.c`? This way we could easily add others like `stmclk_f01.c` and so on?!

Reply:
also I think this needs a rebase, as there is already a `stmclk.c` here containing the `lfclk` and `dbp` functions

Reply:
Theses function (lfck, dbp) are in `stmclk_common.c` and #7500 try to unify with F0 and F1 within this same file